Contentful.Core.Tests/Images/ImageUrlBuilderTests.cs

```diff
@@ -59,6 +59,17 @@ public void SettingJpgQualityShouldSetCorrectQueryString(int quality)
             Assert.Equal($"?q={quality}", result);
         }
 
+        [Fact]
+        public void Setting8BitPngShouldSetCorrectQueryString()
+        {
+            //Arrange
+            var builder = new ImageUrlBuilder();
+            //Act
+            var result = builder.Set8BitPng().Build();
+            //Assert
+            Assert.Equal($"?fl=png8", result);
```

Comment:
it should be `?fm=png&fl=png8`, right?https://www.contentful.com/developers/docs/references/images-api/#/reference/changing-formats/8-bit-pngs/retrieve-an-image/

Reply:
Oh, it actually throws an error, even if the image is already a png, that's bad... I'll fix.